test/suites/remote.sh

```diff
@@ -112,7 +112,7 @@ test_remote_url_with_token() {
   lxc config trust rm "$(lxc config trust list -f json | jq -r '.[].fingerprint')"
 
   # Set token expiry to 5 seconds
-  lxc config set core.remote_token_expiry 5S
+  lxc config set core.remote_token_expiry 1S
```

Comment:
The comment here is outdated now as well.

Reply:
Done!